lib/src/main/java/com/orange/ods/compose/component/banner/OdsBanner.kt

```diff
@@ -115,18 +118,18 @@ fun OdsBanner(
             }
         }
         if (!isSingleLineBanner && buttonCount > 0) {
-            Row(
+            FlowRow(
                 modifier = Modifier
-                    .padding(bottom = dimensionResource(id = R.dimen.spacing_xs))
                     .padding(horizontal = dimensionResource(id = R.dimen.spacing_m))
                     .align(Alignment.End),
-                horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.spacing_s))
+                horizontalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.spacing_s), Alignment.End),
+                verticalArrangement = Arrangement.spacedBy((-6).dp)
```

Comment:
Should we also add `verticalArrangement = Arrangement.spacedBy((-6).dp` to the `FlowRow` containing buttons in `OdsHorizontalCard` in order to be consistent?